resync/workflows/workflow_predictive_maintenance.py

```
@@ -17,25 +17,27 @@
-    END = "END"
-    class StateGraph:  # type: ignore
-        def __init__(self, *args: Any, **kwargs: Any) -> None: pass
-
-
-# PostgresSaver requires psycopg3 with libpq - fallback to MemorySaver if unavailable
-try:
     from langgraph.checkpoint.postgres import PostgresSaver
-    POSTGRES_SAVER_AVAILABLE = True
+    LANGGRAPH_AVAILABLE = True
 except ImportError:
+    class HumanMessage: pass
```

Ruff (0.15.0)

[error] Multiple statements on one line (colon)

(E701)